examples/nativemodules/Storage.test.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { fireEvent, render, act } from 'react-native-testing-library';
+import StorageView from './StorageView';
+
+/* eslint-disable jest/no-test-callback */
+
+// Mock the react-native-testing-library
+// For this test case, we are interested in mocking just 'getItem' and 'setItem'
+jest.mock('@react-native-community/async-storage', () => {
```

Comment:
It seems that async-storage offeres [their own mock](https://github.com/react-native-community/async-storage/blob/master/docs/Jest-integration.md#using-async-storage-mock). Not sure if it's relevant here though.

Reply:
Actually, it's not. But I wanted to pick an example that shows mocking. I plan on including a link to testing in Async storage inside the docs

Reply:
@mdjastrzebski is right, we shouldn't promote writing custom mocks, if there's an official one provided. Please remove the Storage example entirely